core/templates/dev/head/css/oppia.css

```diff
@@ -2646,6 +2646,36 @@ md-card.preview-conversation-skin-supplemental-card {
   margin: 0;
   padding: 0;
 }
+/* Editor Tutorial Welcome Modal */
+.oppia-welcome-modal .modal-dialog {
```

Comment:
Let's pull this and the modal out into its own directive html file, similar to what we do in e.g. core/templates/dev/head/components/exploration_summary_tile_directive.html. (We're moving away from lumping everything into a big CSS file, and are trying to do things a bit more modularly going forward.)